pkgs/by-name/li/libvisual/package.nix

```diff
@@ -33,26 +34,26 @@ stdenv.mkDerivation rec {
     autoreconfHook
     autoconf-archive
     pkg-config
+    validatePkgConfig
   ];
-  buildInputs = [
-    SDL
-    glib
-  ];
+  buildInputs = [ glib ] ++ lib.optional (withLvTool) SDL_compat;
 
   configureFlags =
-    lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-      # Remove when 0.5.x is published.
-      "--disable-lv-tool"
+    [
+      (lib.enableFeature withLvTool "lv-tool")
```

Comment:
Honestly do we even need lvtool? Can we just like, not build it? Or maybe even yeet this entire package? Probably not the best place to have this conversation, but it feels like a lot of complexity for not a lot of gain.